docs/how-to/data-hub/add-data.md

```diff
@@ -0,0 +1,43 @@
+# Add data to a Deep Origin Database
+
+This document describes how to add data to a Deep Origin Database.
+
+Consider the following dataframe constructed from a database using:
+
+```python
+from deeporigin.data_hub import api
+df = api.get_dataframe("xy")
+df
+```
+
+![](../../images/df-xy.png)
+
+## Add new rows
+
+To add new rows to the underlying database, use the `add_databse_rows` function:
+
+```python
+data = dict(X=[1, 2], Y=[2, 3])
+api.add_database_rows(database_id="xy", data=data)
```

Comment:
Do people tend to think of adding data by columns or rows? I would've leaned more towards the latter.